var/spack/repos/builtin/packages/babelstream/package.py

```diff
@@ -68,14 +92,28 @@ class Babelstream(CMakePackage, CudaPackage, ROCmPackage):
         msg="CUDA requires architecture to be specfied by cuda_arch=",
     )
     variant("mem", values=str, default="DEFAULT", description="Enable MEM Target for CUDA")
-    # Raja Conflict
+    # Raja offload
     variant(
-        "offload", values=str, default="none", description="Enable RAJA Target [CPU or NVIDIA]"
-    )
-    conflicts(
-        "offload=none",
+        "raja_offload",
+        values=("cpu", "nvidia"),
+        default="cpu",
         when="+raja",
-        msg="RAJA requires architecture to be specfied by target=[CPU,NVIDIA]",
+        description="Enable RAJA Target [CPU or NVIDIA] / Offload with custom settings for OpenMP",
+    )
+    #std-* offload
+    variant(
+        "std_offload",
+        values=("nvhpc","none"),
+        default="none",
+        when="+std",
+        description="Enable RAJA Target [CPU or NVIDIA] / Offload with custom settings for OpenMP",
+    )
+    variant(
+        "std_use_tbb",
+        values=("on","off"),
+        default="off",
```

Comment:
The issue was we were checking the `models` with boolean ( cuda, acc, omp etc. ) but since you mentioned this I thought it would be better to create a list of models and neglect anything except those models. So in the new commit it is now boolean and the `-DMODEL` filtering is more precise and less crowded to handle different scenarios